src/app/pizzadata.json

```diff
@@ -50,4 +50,4 @@
         "price": 239,
         "img":"Peppersvennen.jpg"
     } 
-]
```

Comment:
Important!

Are you sure this one should be removed?

Reply:
Didnt intentionally change pizzadata. Will revert.